app/models/segmentation_statistic.rb

```diff
@@ -19,6 +19,19 @@ def self.stats_generated_for_day?( at_time = Time.now )
     SegmentationStatistic.where( "DATE(created_at) = DATE(?)", at_time.utc ).exists?
   end
 
+  def self.generate_segmentation_int( start_date, end_date, use_database: false )
```

Comment:
What does "int" mean here? If that's an abbreviation for something can we please use the full word so the meaning of this method is clear?